libs/payments/ui/src/lib/utils/helpers.ts

```diff
@@ -73,23 +73,7 @@ export function formatPlanPricing(
  *   'monthly' or 'months'
  *   'yearly' or 'years'
  * @param interval
- * @param intervalCount
  */
-export function formatPlanInterval({
-  interval,
-  intervalCount,
-}: {
-  interval: PriceIntervalType; // TODO - Replace once FXA-7507 lands
-  intervalCount?: number;
-}): string {
-  switch (interval) {
-    case 'day':
-      return intervalCount === 1 ? 'daily' : 'days';
-    case 'week':
-      return intervalCount === 1 ? 'weekly' : 'weeks';
-    case 'month':
-      return intervalCount === 1 ? 'monthly' : 'months';
-    case 'year':
-      return intervalCount === 1 ? 'yearly' : 'years';
-  }
+export function formatPlanInterval(interval: string): string {
```

Comment:
Could you please update this doc string of this function to match its new functionality?